### PR TITLE
movie-api

### PR DESCRIPTION
**Pull request by muhactive**

### CODE REVIEW

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
don't work on the master branch - it's a bad practice
https://github.com/mate-academy/py-task-guideline/blob/main/README.md#:~:text=git%20checkout%20%2Db%20develop

**Comment by muhactive** [comment]
when i fix bugs they don't push come here

**Comment by muhactive** [comment]
they push in my personal directory, then i push my work like new work

**Comment by muhactive** [comment]
when i chenges my work

**Comment by muhactive** [comment]
i change my branch and push new work

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,160 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
```
Comment (olena-hul) [inline .gitignore:7]: Your merge requested is still from master branch - push the code to the develop branch and change the source branch to develop

**Review by olena-hul — CHANGES_REQUESTED** [review]

**Comment by muhactive** [comment]
https://github.com/mate-academy/py-movie-api/pull/279

**Comment by muhactive** [comment]
this is my new work https://github.com/mate-academy/py-movie-api/pull/279

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
to change your PR branch from master to develop:
you need to remove your repository from github
fork the task again from mate-academy repository
push your code from local machine to forked github repository
make PR from develop branch

**Pull request closed without merging**